s3/sc-s3-synapse-ra-v1.0.1.yaml

```diff
@@ -0,0 +1,98 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: S3Objects
+Description: >-
+  Synapse S3 Custom Storage
+  (https://docs.synapse.org/articles/custom_storage_location.html)
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+          default: S3 Bucket Configuration
+        Parameters:
+          - BucketName
+      - Label:
+          default: S3 Bucket Policy Configuration
+        Parameters:
+          - SynapseUserName
+          - S3UserARNs
+Parameters:
+  SynapseUserName:
```

Comment:
Supported but the recommended value is now Synapse userId or teamId.
https://docs.synapse.org/articles/custom_storage_location.html

Reply:
That's a separate issue.  Would you mind creating a jira issue and we can address it in a separate PR.

Reply:
Sure.